ciphers/fractionated_morse_cipher.py

```diff
@@ -0,0 +1,173 @@
+"""
+Python program for the Fractionated Morse Cipher.
+
+The Fractionated Morse cipher first converts the plaintext to morse code,
+then enciphers fixed-size blocks of morse code back to letters.
+This procedure means plaintext letters are mixed into the ciphertext letters,
+making it more secure than substitution ciphers.
+
+For more information visit - http://practicalcryptography.com/ciphers/fractionated-morse-cipher/
+
+"""
+
+
+import string
+
+# Define Morse code dictionary
+MORSE_CODE_DICT = {
+    "A": ".-",
+    "B": "-...",
+    "C": "-.-.",
+    "D": "-..",
+    "E": ".",
+    "F": "..-.",
+    "G": "--.",
+    "H": "....",
+    "I": "..",
+    "J": ".---",
+    "K": "-.-",
+    "L": ".-..",
+    "M": "--",
+    "N": "-.",
+    "O": "---",
+    "P": ".--.",
+    "Q": "--.-",
+    "R": ".-.",
+    "S": "...",
+    "T": "-",
+    "U": "..-",
+    "V": "...-",
+    "W": ".--",
+    "X": "-..-",
+    "Y": "-.--",
+    "Z": "--..",
+    " ": "",
+}
+
+# Define possible trigrams of Morse code
+MORSE_COMBINATIONS = [
+    "...",
+    "..-",
+    "..x",
+    ".-.",
+    ".--",
+    ".-x",
+    ".x.",
+    ".x-",
+    ".xx",
+    "-..",
+    "-.-",
+    "-.x",
+    "--.",
+    "---",
+    "--x",
+    "-x.",
+    "-x-",
+    "-xx",
+    "x..",
+    "x.-",
+    "x.x",
+    "x-.",
+    "x--",
+    "x-x",
+    "xx.",
+    "xx-",
+    "xxx",
+]
+
+# Create a reverse dictionary for Morse code
+REVERSE_DICT = {value: key for key, value in MORSE_CODE_DICT.items()}
+
+
+def encode_to_morse(plaintext: str) -> str:
+    """Encode a plaintext message into Morse code.
+
+    Args:
+        plaintext (str): The plaintext message to encode.
```

Comment:
Please do not repeat the datatypes that are checked by mypy in the function signature because users will be confuse if one is modified and the other is not.  Repeat for other function docstrings.
```suggestion
        plaintext: The plaintext message to encode.
```